dist/completer.js

```diff
@@ -57,8 +57,7 @@ System.register(['lodash'], function(exports_1) {
                                 return;
                             }
                             this.getFilterKeyAndValueForMetric(metricType).then(function (result) {
-                                // to filter query, need to use 'label'
-                                var valuePicker = lodash_1.default.property(filterKey.replace(/label/g, 'labels'));
```

Comment:
Why was this ever in here?

Reply:
I believe there was a bug in stackdriver API that for filter they expected "label" instead of "labels" and for everything else it was "labels".

Right now it works for both "label" and "labels" so it seems they have fixed the issue.